src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.BuildTableDataSource.cs

```diff
@@ -193,6 +193,9 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         case StandardTableKeyNames.ErrorCodeToolTip:
                             content = GetHelpLinkToolTipText(data);
                             return content != null;
+                        case StandardTableKeyNames.HelpKeyword:
```

Comment:
@CyrusNajmabadi some of them could, but I dont think it is that bad to require cleaning up.